ci/travis/build-docker-images.sh

```diff
@@ -16,6 +17,45 @@ docker_push() {
         echo "Skipping docker push because it's in PR environment."
     fi
 }
+build_and_push_tags() {
+    # $1 image-name (Dockerfile directory)
```

Comment:
Can we clearing this comment a little bit. I think you meant something like: image-name, we will also use this as the build directory for docker. So `$1=base-deps` means image will be called base-deps and we will build docker image in base-deps directory.